src/kakarot/accounts/library.cairo

```diff
@@ -493,6 +523,44 @@ namespace Internals {
         return ();
     }
 
+    // @notice Store the jumpdests of the contract.
+    // @param index The current free index in the jumpdests_ storage.
+    // @param jumpdests_len The length of the jumpdests.
+    // @param jumpdests The jumpdests of the contract.
+    func write_jumpdests{syscall_ptr: felt*}(jumpdests_len: felt, jumpdests: felt*) {
+        alloc_locals;
+
+        if (jumpdests_len == 0) {
+            return ();
+        }
+
+        // selector!("Account_valid_jumpdests")
+        local base_address = 0x01eb4a0d1c10536edc8183a507de5f884c0a18b4877da7c986a6100d6bfda343;
```

Comment:
so this is `Account_valid_jumpdests.addr();` ?

Reply:
yes but if I use `.addr()` I need to manually handle `pedersen_ptr` and `range_check_ptr` and re-assign them at each iteration for nothing

Reply:
indeed, or put them in locals to be able to rebind at the end.
we can stay like that, but then plz update the comment from cairo1 style to

```
// hardcoded value of Account_valid_jumpdests.addr() to avoid rebinding ...
```